test/Microsoft.ML.TestFrameworkCommon/Datasets.cs

```diff
@@ -98,6 +98,19 @@ public static class TestDatasets
             extraSettings = "/cacheinst- /inst Pipe{loader=Text{sparse- col=Attr:TX:6 col=Label:0 col=Features:1-5,6,7-9} lab=Label feat=Features}"
         };
 
+        /// <summary>
+        /// Modified version of breastCancerPipe without mamlExtraSettings.
+        /// </summary>
+        public static TestDataset breastCancerPipeWithoutMamlExtraSettings = new TestDataset
+        {
+            name = "breast-cancer",
+            trainFilename = "breast-cancer.txt",
+            testFilename = "breast-cancer.txt",
+            // Using "col=Features:1-5,6,7-9" improves code coverage. Same with "col=Attr:TX:6".
+            loaderSettings = "loader=Text{sparse- col=Attr:TX:6 col=Label:0 col=Features:1-5,6,7-9}",
+            extraSettings = "/cacheinst- /inst Pipe{loader=Text{sparse- col=Attr:TX:6 col=Label:0 col=Features:1-5,6,7-9} lab=Label feat=Features}"
```

Comment:
Should be able to remove (background: https://github.com/dotnet/machinelearning/issues/4742)

```suggestion
```